### ISSUE TITLE
Reading configuration of devices in a run doesn't work with tiled server

### ISSUE THREAD

**Issue by whs92** [bluesky/databroker#745]
## Expected Behavior
With databroker backed by intake I am able to access the configuration parameters of devices like this:

```python
db = databroker.catalog['xyz']
run = db[-1]
run.primary.config['motor'].read()
```
This gives an xarray of device configuration parameters. I would like the same functionality using a databroker with a tiled server like this:

```python
from tiled.client import from_uri
db = from_uri("http://localhost:8000/api")
run = db[-1]
run.primary.config['motor'].read()
```

## Current Behavior
At the moment I get an error something like:
```
ClientError: 404: No such entry: ['a6c2c56d-cbea-40a0-a83f-bea66cc41552', 'baseline', 'config', 'ue48_m3'] http://172.17.9.22:8001/api/node/metadata/a6c2c56d-cbea-40a0-a83f-bea66cc41552/baseline/config/ue48_m3
```

## Steps to repeat

```python
from bluesky import RunEngine
from databroker.v2 import temp

## Set up env


RE = RunEngine({})
db = temp()
RE.subscribe(db.v1.insert)

RE(scan([det],motor,1,2,10))
db[-1].primary.config['det'].read()
```

## Context
I use the configuration parameters of the devices involved in a previous plan to restore the configuration if I want to repeat the plan. The ability to get to this through databroker is really useful.

## Your Environment
I am using:

```
tiled==0.1.0a77
databroker==2.0.0b11
```

**Comment by whs92** [comment]
If it helps, the configuration data is available when I access it through the uri and look at the JSON file. 

`http://172.17.9.22:8001/api/node/metadata/3e0c571e-37f4-4a44-b3f8-9def51066b9f/primary/data`

![image](https://user-images.githubusercontent.com/44977769/204775421-9c4e4bcd-7c9d-4390-a6e2-720c29a19c6a.png)

Interestingly there is nothing in the `primary.config.uri` = `'http://172.17.9.22:8001/api/node/metadata/3e0c571e-37f4-4a44-b3f8-9def51066b9f/primary/config' 

![image](https://user-images.githubusercontent.com/44977769/204775917-6858a860-ba34-4650-a699-ae8da4384d33.png)

**Comment by whs92** [comment]
I can get the data I need with the very convoluted:

`run.primary.data.metadata['descriptors'][0]['configuration']['ue48_pgm_en']['data']`

This is good enough for now.

**Comment by danielballan** [comment]
Thanks for the additional context. This came up at NSLS-II on Monday and I'm hoping to find time to patch it in the next couple days.

**Comment by danielballan** [comment]
This was straightforward to fix fortunately. We had no test coverage on the v2-style configuration access, and so a regression slipped through. Fix in #750.

**Comment by danielballan** [comment]
Fixed released in v2.0.0b13. It is on PyPI now and should be on conda-forge by end-of-day.
